Packages/ohif-measurements/client/components/measurementLightTable/measurementLightTableView/measurementLightTableView.html

```diff
@@ -9,4 +9,21 @@
             {{/each}}
         {{/each}}
     {{/let}}
+    {{#if hasAnyMeasurement}}
+    <div class="report-area">
+        {{#if saveEnabled}}
+            {{#unless noUnsavedChanges}}
```

Comment:
Why do we need the double negative (unless no changes)? Can you change this to "{{# if hasUnsavedChanges }} ?